core/msd/src/main/rdl/KubernetesNetworkPolicy.rdli

```diff
@@ -0,0 +1,23 @@
+// Copyright The Athenz Authors
+// Licensed under the terms of the Apache version 2.0 license. See LICENSE file for terms.
+
+include "Names.tdl";
+include "KubernetesNetworkPolicy.tdl";
+
+// API endpoint to get the Kubernetes network policy converted from the corresponding MSD policy
+resource KubernetesNetworkPolicyResponse GET "/domain/{domainName}/service/{serviceName}/kubernetesnetworkpolicy" {
```

Comment:
Is it feasible to call one service per request, for an environment with hundreds/thousands of services ?

Reply:
It would be good to have an option to make batch call, another endpoint is also fine.

Reply:
yeah that could be another endpoint.